### PR TITLE
CI: Test with RabbitMQ 4.0.x

### PR DESCRIPTION
**Pull request by danielhollas**
Let's see if RMQ 4.0 passes the tests.

https://aiida.discourse.group/t/supported-rabbitmq-versions/392/11?u=danielhollas

### CODE REVIEW

**Review by unkcpz — APPROVED** [review]
The tests all pass, so all good .

File: .docker/aiida-core-base/Dockerfile [hunk 1]
```diff
@@ -24,7 +24,7 @@ ENV SYSTEM_USER="${SYSTEM_USER}"
 
 # Install all OS dependencies for notebook server that starts but lacks all
 # features (e.g., download as all possible file formats)
-ENV DEBIAN_FRONTEND noninteractive
+ENV DEBIAN_FRONTEND=noninteractive
```
[the remark this answers is not in the document]
Comment (unkcpz) [reply .docker/aiida-core-base/Dockerfile:27]: Actually I like if we see tiny things (hard to say how tiny is tiny, for me it is like one line change not related to the code logic and sure it is 100% correct) like this we clean it up.

**Review by unkcpz — CHANGES_REQUESTED** [review]
The nightly test also failed and I guess it is because of https://github.com/aiidateam/aiida-core/pull/6640

**Comment by danielhollas** [comment]
Whoops, RMQ and nightly tests are failing due the uv lockfile PR. I'll fix that.

**Review by unkcpz — APPROVED** [review]

**Pull request merged**